superdesk/publish/publish_queue.py

```diff
@@ -126,6 +129,15 @@ def on_create(self, docs):
                 subscriber = subscriber_service.find_one(req=None, _id=doc['subscriber_id'])
                 doc['published_seq_num'] = subscriber_service.generate_sequence_number(subscriber)
 
+            # if keys are not returned by client (can happen on resend), we need to get them for database
+            keys = {'item_encoding', 'encoded_item_id'}
+            if not keys.issubset(doc):
+                item = get_resource_service("publish_queue").find_one(req=None, item_id=doc['item_id'])
+                if item is not None:
```

Comment:
I didn't knew we could have several item_id, I've tried with pick_fields but it doesn't seems to work. Unfortunately I spend most of my days in meetings, I could really find the origin of the problem, will continue on this tomorrow.